assets/js/components/Header.stories.js

```diff
@@ -144,7 +158,7 @@ HeaderWithCustomHelpMenuLinks.args = {
 export const HeaderWithSubHeader = Template.bind( {} );
 HeaderWithSubHeader.storyName = 'Plugin Header with Sub Header';
 HeaderWithSubHeader.args = {
-	subHeader: <UserInputSuccessBannerNotification />,
+	subHeader: <SubHeaderBannerNotification />,
```

Comment:
Seeing as we're going to be removing Optimize, I didn't replace `UserInputSuccessBannerNotification` with the `OptimizeRemovalNotification` component here, rather created a simple banner component within this test as the other existing notification components required a bit more setup.